Fix for max_ul_rate and max_dl_rate kilobyte conversions

tinamil opened the pull request:
This fixes the webui to allow the user to input the size in kB/s while internally using B/s.  The status bar right-click context menu shows the correct values as well, but the ability to set global rate limits from the status bar appears to already be broken before my patch so that functionality is still broken.

This fixes problem number 2 of hadouken/hadouken#166, it still only allows 6 digits but 999999 is now correctly a 1 GB/s limit instead of a 1 MB/s limit.

tinamil committed:
Fix for max_ul_rate and max_dl_rate to correctly handle byte/kilobyte displays.

This fixes the webui to allow the user to input the size in kB/s while internally using B/s.  The status bar right-click context menu shows the correct values as well, but the ability to set global rate limits from the status bar appears to already be broken before my patch so that functionality is still broken.

This fixes problem number 2 of hadouken/hadouken#166

vktr commented on webui.js:
```diff
@@ -2245,7 +2245,10 @@ var utWebUI = {
                     break;
-
+				case "max_dl_rate":
+				case "max_ul_rate":
```
Change tabs to spaces.

vktr commented on webui.js:
```diff
@@ -2597,7 +2604,7 @@ var utWebUI = {
             }
         }
-
+		
```
Unnecessary tab.

vktr commented:
I've left a few super minor style comments, otherwise you're good to go!

tinamil committed:
Fixed style issues

Removed unnecessary tabs and changed tabs to spaces to match the rest of the file.
